lib/autorest/powerBIClient.d.ts

```diff
@@ -4,7 +4,7 @@
  * regenerated.
  */
 
-import { ServiceClientCredentials, ServiceClientOptions, RequestOptions, ServiceCallback } from 'ms-rest';
```

Comment:
There is a bug in autorest where `ServiceClientCredentials` is not imported.  Make sure this goes back in.